### ISSUE TITLE
Linting for yaml and toml file

### ISSUE THREAD

**Issue by VascoSch92** [astral-sh/ruff#10227]
Hey 👋 

If I'm correct, there are not rules which target `.yaml` or `.toml` config files.

- Could be interesting to format also these types of files?
- Is this already a things?

**Comment by MichaReiser** [comment]
Yeah that would be neat. It's not our primary focus for now. There's still a lot to do on the Python front. 

Related to https://github.com/astral-sh/ruff/issues/9244

**Comment by Kilo59** [comment]
@ashrub-holvi 

I agree it would be nice if ruff could format/lint toml + yaml but it is possible to use `prettier` for this without requiring node to be a project dependency. Just use the [pre-commit](https://prettier.io/docs/en/precommit#option-2-pre-commithttpsgithubcompre-commitpre-commit) hook and the [pre-commit](https://pre-commit.ci/).


Example 
https://github.com/great-expectations/great_expectations/blob/200c968b5b5c6b19d5d2d63ca0d6fe1835b94b62/.pre-commit-config.yaml#L27-L32

**Comment by ashrub-holvi** [comment]
Not used yet, but found prettier for YAML written in Rust https://github.com/g-plane/pretty_yaml

**Comment by ashrub-holvi** [comment]
I guess this issue can be closed as duplicate, main discussion is going in https://github.com/astral-sh/ruff/issues/10738 and it's mostly about using [dprint](https://dprint.dev/).

**Comment by MichaReiser** [comment]
I think it's different as it's about linting and not formatting (although it does mention formatting..)
